src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -0,0 +1,98 @@
+#include "effects/backends/builtin/pitchshifteffect.h"
+
+#include "util/sample.h"
+
+PitchShiftGroupState::~PitchShiftGroupState() noexcept {
+    SampleUtil::free(m_retrieveBuffer[0]);
+    SampleUtil::free(m_retrieveBuffer[1]);
+
+    VERIFY_OR_DEBUG_ASSERT(m_pRubberBand) {
+        return;
+    }
+
+    m_pRubberBand->reset();
+}
+
+// static
+QString PitchShiftEffect::getId() {
+    return "org.mixxx.effects.pitchshift";
```

Comment:
I think you mean `QStringLiteral`

Reply:
I used `QString` because all other built-in effects use it too.